components/tooltip/tooltip.js

```diff
@@ -578,12 +579,23 @@ class Tooltip extends RtlMixin(LitElement) {
 	}
 
 	_computeSpaceAround(offsetParent, targetRect) {
-		const spaceAround = {
+
+		const boundingContainer = getBoundingAncestor(this);
+		const bounded = (boundingContainer !== document.documentElement);
+		const boundingContainerRect = boundingContainer.getBoundingClientRect();
+
+		const spaceAround = (bounded ? {
```

Comment:
This is almost the same as dropdown, except the treatment of margins is different.